<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1523</width>
    <height>911</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>637</width>
    <height>454</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>1</width>
    <height>1</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>30</y>
     <width>87</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferAntialias</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>SizeHorCursor</cursorShape>
   </property>
   <property name="text">
    <string>Owner</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>288</x>
     <y>433</y>
     <width>143</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Licence Validity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>288</x>
     <y>470</y>
     <width>162</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Insurance Validity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>288</x>
     <y>510</y>
     <width>133</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Emmision Test</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>470</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>510</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>282</y>
     <width>63</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Name       </string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>330</y>
     <width>46</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Adress</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>420</y>
     <width>25</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>NIC</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>460</y>
     <width>97</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Licence Expire</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>500</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Blood Group</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>340</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Vehicle Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>288</x>
     <y>29</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferAntialias</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>SizeHorCursor</cursorShape>
   </property>
   <property name="text">
    <string>Vehicle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadbutton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>300</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="imagelabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>441</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Karumbi</family>
     <pointsize>40</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>VEHICLE NUMBER</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_2">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>110</y>
     <width>130</width>
     <height>130</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>256</width>
     <height>192</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="vehicleNumbInput">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>330</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="submitbutton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>370</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="checkVehicleNumb">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>330</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Lname">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>280</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Fname">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>280</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="line1">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>330</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="line2">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>350</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="line3">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>370</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nic">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>420</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="expDate">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>460</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="blood">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>500</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar1">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>430</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>636</x>
     <y>560</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QComboBox" name="location">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>570</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>KOTTWA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GALLE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MATARA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>570</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ENTRANCE</string>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>progressBar2</zorder>
  <zorder>progressBar3</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>label_3</zorder>
  <zorder>label</zorder>
  <zorder>textBrowser_2</zorder>
  <zorder>loadbutton</zorder>
  <zorder>imagelabel</zorder>
  <zorder>vehicleNumbInput</zorder>
  <zorder>submitbutton</zorder>
  <zorder>checkVehicleNumb</zorder>
  <zorder>Lname</zorder>
  <zorder>Fname</zorder>
  <zorder>line1</zorder>
  <zorder>line2</zorder>
  <zorder>line3</zorder>
  <zorder>nic</zorder>
  <zorder>expDate</zorder>
  <zorder>blood</zorder>
  <zorder>progressBar1</zorder>
  <zorder>nextButton</zorder>
  <zorder>location</zorder>
  <zorder>label_12</zorder>
 </widget>
 <tabstops>
  <tabstop>textBrowser_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
